trackers/javascript-tracker/test/integration/media.test.ts

```diff
@@ -0,0 +1,371 @@
+  let out = {
+    context: [
+      {
+        schema: 'iglu:org.whatwg/media_element/jsonschema/1-0-0',
+        data: {
+          htmlId: htmlId,
+          mediaType: 'VIDEO',
+          autoPlay: false,
+          buffered: [{ start: 0, end: jasmine.any(Number) }],
+          controls: true,
+          crossOrigin: null,
+          currentSrc: 'http://snowplow-js-tracker.local:8080/test-video.mp4',
+          defaultMuted: true,
+          defaultPlaybackRate: 1,
+          disableRemotePlayback: false,
+          error: null,
+          networkState: jasmine.stringMatching('/NETWORK_(EMPTY|IDLE|LOADING|NO_SOURCE)/'),
+          preload: jasmine.stringMatching('/auto|metadata|none|/'),
+          readyState: jasmine.stringMatching('/HAVE_(NOTHING|METADATA|CURRENT_DATA|FUTURE_DATA|ENOUGH_DATA)/'),
+          seekable: [{ start: 0, end: jasmine.any(Number) }],
+          seeking: false,
+          src: 'http://snowplow-js-tracker.local:8080/test-video.mp4',
+          textTracks: [],
+          fileExtension: 'mp4',
+          fullscreen: false,
+          pictureInPicture: false,
+        },
+      },
+      {
+        schema: 'iglu:com.snowplowanalytics.snowplow/media_player/jsonschema/1-0-0',
+        data: {
+          currentTime: jasmine.any(Number),
+          duration: 20,
+          ended: false,
+          loop: false,
+          muted: true,
+          paused: false,
+          playbackRate: 1,
+          volume: 100,
+          ...mediaPlayerSchemaExpected,
+        },
+      },
+      {
+        schema: 'iglu:org.whatwg/video_element/jsonschema/1-0-0',
+        data: {
+          autoPictureInPicture: false,
+          disablePictureInPicture: false,
+          poster: '',
+          videoHeight: 144,
+          videoWidth: 176,
+        },
+      },
+    ],
+    unstruct_event: {
+      schema: 'iglu:com.snowplowanalytics.snowplow/unstruct_event/jsonschema/1-0-0',
+      data: {
+        schema: 'iglu:com.snowplowanalytics.snowplow/media_player_event/jsonschema/1-0-0',
+        data: { type: eventType, label: 'test-label' },
+      },
+    },
+  };
+  return out;
+};
+
+const compare = (expected: any, received: any) => {
+  for (let i in expected.context.length) {
+    expect(expected.context[i].schema).toEqual(received.event.contexts.data[i].schema);
+    expect(expected.context[i].data).toEqual(jasmine.objectContaining(received.event.contexts.data[i].data));
+  }
+  expect(expected.unstruct_event).toEqual(received.event.unstruct_event);
+};
+
+let docker: DockerWrapper;
+let log: Array<unknown> = [];
+
+describe('Media Tracker', () => {
+  const getFirstEventOfEventType = (eventType: string) => {
+    let results = log.filter((l: any) => l.event.unstruct_event.data.data.type === eventType);
+    return results[results.length - 1];
+  };
+
+  if (browser.capabilities.browserName === 'internet explorer') {
+    it.only('Skip IE9', () => true);
+    return;
+  }
```

Comment:
Does this end up skipped all Internet Explorers? I feel like this should probably work in at least IE11.